library/src/main/java/com/google/android/exoplayer2/text/ssa/Style.java

```diff
@@ -0,0 +1,283 @@
+package com.google.android.exoplayer2.text.ssa;
+
+import java.util.Map;
+
+import static android.R.attr.angle;
+
+/**
+ * Created by cablej01 on 27/12/2016.
+ */
+
+public class Style {
```

Comment:
Sounds good. I just wanted a branch point where everything was captured and titles would render without style. Happy to change the approach to fit the architecture.